packages/amazonq/.changes/next-release/Feature-73eb1158-43e8-4d5f-86e4-d245dc2e4af1.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "add modifyDefaultSource folder event when customer uses /dev source folder modification feature"
```

Comment:
please review https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

this is not a customer-impacting change

Reply:
Thanks, I will remove this.